python/README.md

```diff
@@ -9,6 +9,22 @@ aim is to provide canonical examples of various mechanics available in Flyte, an
  - How to accept inputs and produce outputs from a task and a workflow.
  - How to use complex datatypes like - Schemas, Blobs and CSVs.
 
+## Run these examples in an existing Flyte Installation
+NOTE: these workflows are written and configured to run in Sandbox mode (Minio instead of using cloud blob stores). But, this can be easily overridden. Follow the steps
+
+ ### Register the examples in Flyte Sandbox environment
```

Comment:
not clear what is happening.  Are these commands supposed to work on a local docker-for-desktop deployment?  or a cloud deployment?  Why is the platform url 127.0.0.1:30081?

Reply:
you are right, I should change that to a "<template>"